vision_agent/lmm/lmm.py

```diff
@@ -449,7 +446,7 @@ def generate(
         ]
         if media:
             for m in media:
-                encoded_media = encode_media(m)
+                encoded_media = encode_media(m, resize=768)
```

Comment:
Is that tht maximum resize available ?

Reply:
It's not exactly the maximum but close to it. They don't do any resizing on their end